js/main.js

```diff
@@ -1,70 +1,90 @@
+var player;
+
 window.onload = function() {
+	console.log("setup player");
 	setupPlayer();
 	setupControllerEvents();
 }
 
-
 function setupPlayer() {
-	var conf = {
-		key : "<YOUR_PLAYER_KEY>",
-		source : {
-			// AVC Stream
-			//dash : "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd",
-			// HEVC Stream
-			//dash : "https://bitmovin-a.akamaihd.net/content/multi-codec/hevc/stream.mpd"
-		
-			//Widevine Stream
-			dash: "http://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd",
-			drm: {
-				widevine: {
-					LA_URL: "https://widevine-proxy.appspot.com/proxy",
-				}			
-			}
-		},
-		playback : {
-			autoplay : true
-		},
-		tweaks : {
-			file_protocol : true,
-			app_id : "com.bitmovin.demo.webapp"
-		}
-	};
+	var config = {
+			  key: '7a94a7fd-61a6-40e1-b357-3d19e33ebcb1',
```

Comment:
please remove keys from open source repos.

Reply:
yeah, hit me during the night that I pushed with it, sorry.